packages/fx-core/src/plugins/resource/bot/constants.ts

```diff
@@ -259,5 +259,5 @@ export class TelemetryValues {
 }
 
 export class AzureConstants {
-  public static readonly requiredResourceProviders = ["Microsoft.Web", "Microsoft.Storage"];
+  public static readonly requiredResourceProviders = ["Microsoft.Web", "Microsoft.Storage", "Microsoft.BotService"];
```

Comment:
Have you confirmed that we need Microsoft.Storage?

Reply:
tested, don't need